sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/http/URLHelper.java

```diff
@@ -0,0 +1,45 @@
+package com.mesosphere.sdk.dcos.http;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+/**
+ * URLHelper for easier URL building.
+ */
+public class URLHelper {
+
+    public static URL fromUnchecked(String url) {
+
+        try {
+            return new URL(url);
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
```

Comment:
This meant to be a common helper where developers can decide whether they trust that URL is correct and that exception won't be raised while appending a path to a URL.